.github/actions/set-gha-refs/Set-Called-GHA-Refs.ps1

```diff
@@ -0,0 +1,35 @@
+param(
+    [String[]] $CalledRepoBaseIncludeList,
+    [String[]] $PathIncludeList,
+    [String[]] $FileIncludeList,
+    [String] $ExpectedRef
+)
+
+if ($CalledRepoBaseIncludeList.Count -eq 0)
+{
+    Write-Output '::warning file=Check-Called-GHA-refs.ps1,line10::CalledRepoBaseIncludeList is empty which is unexpected. If this was intentional, you can ignore this warning.'
+    exit 0 # Nothing to check because array is empty.
+}
+
+$CalledRepoBaseIncludeList = $CalledRepoBaseIncludeList.ForEach({ 'uses:\s*' + $PSItem + '(.*)@(.*)' })
```

Comment:
Based on [GitHub's naming restrictions](https://docs.github.com/en/get-started/using-git/dealing-with-special-characters-in-branch-and-tag-names#naming-branches-and-tags), I don't see any overlap with regex special characters, so I don't think this will be an issue.